shaders/src/light_indirect.fs

```diff
@@ -592,6 +591,16 @@ void evaluateIBL(const MaterialInputs material, const PixelParams pixel, inout v
     vec3 E = vec3(0.0); // TODO: fix for importance sampling
     Fr = isEvaluateSpecularIBL(pixel, shading_normal, shading_view, shading_NoV);
 #endif
+
+    SSAOInterpolationCache interpolationCache;
+#if defined(BLEND_MODE_OPAQUE) || defined(BLEND_MODE_MASKED)
+    interpolationCache.uv = uvToRenderTargetUV(getNormalizedViewportCoord().xy);
+#endif
```

Comment:
That seems a bit dangerous -- could we remove the `uv` field from `SSAOInterpolationCache` when `defined(BLEND_MODE_OPAQUE) || defined(BLEND_MODE_MASKED)` is not true?